README.md

```diff
@@ -0,0 +1,26 @@
+# Archway Netowrks
+
+This repo contains a chain.json and assetlist.json for Archway Protocol based networks. A chain.json contains data that makes it easy to start running or interacting with a node.
+
+Schema files containing the recommended metadata structure can be found in the *.schema.json files located in the root directory. Schemas are still undergoing revision as user needs are surfaced. Optional fields may be added beyond what is contained in the schema files.
+
+# Adding new IBC paths
+
+If you are a relayer operator and wish to add a new IBC path connecting to Archway Networks, please follow the below mentioned criteria and create a PR after creating the path.
+## Requirements for the IBC channel:
+- Archway client's trusting period should be set to 2/3 of unbonding period e.g. for archway-1 (mainnet) the clients trusting period must be 1209600s
```

Comment:
Perhaps I would reword this differently, like talking chain A/B or source/destination since there can be relayers that don't have their 'own chain' as the wording currently suggests